mmcv/image/io.py

```diff
@@ -137,9 +139,16 @@ def _pillow2array(img, flag='color', channel_order='bgr'):
     return array
 
 
-def imread(img_or_path, flag='color', channel_order='bgr', backend=None):
+def imread(img_or_path,
+           flag='color',
+           channel_order='bgr',
+           backend=None,
+           file_client_args=None):
     """Read an image.
 
+    Note:
+        In v1.3.17 and later, add `file_client_args` parameters.
```

Comment:
Maybe v1.3.17 --> v1.3.19

Reply:
Yes, I will modify this problem in the next commit.